cli/src/function.rs

```diff
@@ -256,6 +259,30 @@ impl Function {
     }
 }
 
+/// Parse current project code
+/// and return all functions encountered with `kinetics` attributes.
```

Comment:
@quesabe macro or attributes?

Reply:
Correct, the macro is what I meant. Fixed it.